apps/api/tasks.py

```diff
@@ -43,8 +43,6 @@ def setup_connect_channels_for_bots(self, connect_id: UUID, experiment_data_map:
 
     connect_client = CommCareConnectClient()
 
-    # TODO: Refactor when experiment_id is directly on the ParticipantData table
```

Comment:
I tried prefetching the experiment channel as well and doing
```python
channel = experiment.experimentchannel_set.get(platform=ChannelPlatform.COMMCARE_CONNECT)
```
in the for loop, but through inspecting the number of DB queries that we then run I can see the "refactored" version runs more queries than this version, so I'm leaving this as-is for now.